Framework/DataHandling/test/LoadPSIMuonBinTest.h

```diff
@@ -92,7 +92,12 @@ class LoadPSIMuonBinTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(ws->y(0).size(), 10240);
     TS_ASSERT_EQUALS(ws->e(0).size(), 10240);
 
-    TS_ASSERT_DELTA(ws->x(0)[0], -0.160, 0.001);
+    // Check that each spectra is shifted by the correct time zero
+    TS_ASSERT_DELTA(ws->x(0)[0], -0.1582, 0.001);
+    TS_ASSERT_DELTA(ws->x(1)[0], -0.1553, 0.001);
+    TS_ASSERT_DELTA(ws->x(2)[0], -0.1592, 0.001);
+    TS_ASSERT_DELTA(ws->x(3)[0], -0.1602, 0.001);
+
```

Comment:
shouldn't one of these have come out the same as the original?

Reply:
The last one is the same as the original just to one more significant figure. In any case I am going to undo the change where I shift each spectra of the data by the correct time zero for now because as you point out below that needs to be done at the same time as the GUI update.